.github/workflows/source.yml

```
@@ -13,8 +13,16 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  style:
+  lint:
+    runs-on: ubuntu-22.04
+
+    steps:
+    - name: check WarpX python code
```

I forgot to get the source code:
```suggestion
    - uses: actions/checkout@v3
    - name: check WarpX python code
```